pkg/operator/ceph/csi/template/rbd/csi-rbdplugin-provisioner-dep.yaml

```diff
@@ -170,6 +170,10 @@ spec:
               mountPath: /etc/ceph-csi-config/
             - name: keys-tmp-dir
               mountPath: /tmp/csi/keys
+            {{ if .MountCustomCephConf }}
```

Comment:
Would the rbd or cephfs plugins or provisioners ever need different ceph.conf?

Reply:
we haven't come across any such cases. till now we are using the same ceph.conf for both rbd and cephfs in cephcsi also